tools/mpremote/mpremote/commands.py

```diff
@@ -300,6 +300,19 @@ def _mkdir(a, *b):
         do_filesystem_cp(state, src_path_joined, dest_path_joined, False, check_hash)
 
 
+def do_filesystem_recursive_rm(state, path, args):
+    if state.transport.fs_isdir(path):
+        for entry in state.transport.fs_listdir(path):
+            do_filesystem_recursive_rm(state, _remote_path_join(path, entry.name), args)
+        if path not in ["/"]:
```

Comment:
This can just be `if path != "/"` (although maybe this check is not needed at all, if it should print an error about unable to remove root).

Reply:
If we want to support `mpremote rm -r :` then this check just needs to be `if path:`.